### DISCUSSION TITLE
Help with Parse Error when Importing Dialogic

### DISCUSSION THREAD

**Discussion by notomm** [dialogic-godot/dialogic#2155] [Q&A] [answered]
I'm gonna go crazy. I tried enabling the plugin and restarting once, twice... I even moved the .godot folder to force a reimport of the contents, but still nothing, I still get parse error on all the autoloads of dialogic. Dialogic would be perfect for my project but I'm so mad I can't even open it. I tried reading all the documentation and looked at all the releases. I did everything but still can't understand. can anyone help me?

**Comment by CakeVR** [comment, marked as answer]
Hello, you are not providing much information.
Make sure, as stated in the Documentation, you require Godot 4.2.

Let's clear your project.
Delete the Dialogic folder, go to Project -> Reload Current Project.

Now, I recommend using the `main` branch, it has a lot of fixes.

Do this to get the branch:
https://docs.dialogic.pro/getting-started.html#-how-to-install-the-latest-wip-version-

After this, do: https://docs.dialogic.pro/getting-started.html#1-installation--activation

Now, reload the current project again or restart Godot as instructed. Let me know if this works.

**Reply by notomm** [reply]
yes, thank you. I'm so stupid I was sure I upgraded my Godot to 4.2 but I was still on 4.1. Thank you so much for helping me.

**Reply by CakeVR** [reply]
I'm glad it works for you now!
